compiler-core/src/javascript/tests/numbers.rs

```diff
@@ -88,6 +88,28 @@ fn go() {
     );
 }
 
+#[test]
+fn float_operator_complex_expr() {
+    assert_js!(
+        r#"
+fn go() {
+  case 1.0 >=. 0.0 {
+    True -> 2.0
+    False -> 4.0
+  } /. 2.0 // => 1.0
+  case 1 >= 0 {
+    True -> 2
+    False -> 4
+  } / 2 // => 1
+  case 1 >= 0 {
+    True -> 2
+    False -> 4
+  } % 2 // => 0
+}
+"#,
```

Comment:
Only on thing being tested per test please

Gleam uses `->` rather than `=>` too!

Reply:
I copied the output comment from the test above. want me to swap those ones?

Reply:
though tbh the output comment doesn't actually matter here since we're just checking on the generated code so maybe i'll just remove it